x/dualstaking/keeper/delegate.go

```diff
@@ -0,0 +1,572 @@
+package keeper
+
+// Delegation allows securing funds for a specific provider to effectively increase
+// its stake so it will be paired with consumers more often. The delegators do not
+// transfer the funds to the provider but only bestow the funds with it. In return
+// to locking the funds there, delegators get some of the provider’s profit (after
+// commission deduction).
+//
+// The delegated funds are stored in the module's NotBondedPoolName account. Upon
```

Comment:
delegators can delegate or unbond. so shouldn't delegated funds be in the BondedPool? (and not the NotBondedPool)

Reply:
fixed  96a898ea